mm2src/coins/utxo/utxo_common.rs

```diff
@@ -1299,8 +1267,9 @@ where
         coin.as_ref().signature_version,
         coin.as_ref().fork_id
     ));
+    let fee_amount = data.fee_amount + data.change;
```

Comment:
Until recently, if the `change` less than dust, it was added to the `fee` otherwise a change output was added.
Now if the `change` more than dust, it is set to zero and a change output is added.
I agree the `change: u64` field name confuses. How about `unused_change: Option<u64>`?